File: my.py
# Использование встроенного словаря
student = {}  # создание хеш-таблицы

# Добавление элементов
student["name"] = "Alice"
student["age"] = 20
student["grade"] = "A"

# Получение значений
print(student["name"])    # Alice
print(student.get("age")) # 20

# Проверка наличия ключа
if "grade" in student:
    print("Есть оценка")

# Удаление
del student["grade"]

# Перебор всех элементов
for key, value in student.items():
    print(f"{key}: {value}")

File: my.java
import java.util.HashMap;

public class SimpleHashTable {
    public static void main(String[] args) {
        // Создание хеш-таблицы
        HashMap<String, String> student = new HashMap<>();
        
        // Добавление элементов
        student.put("name", "Alice");
        student.put("age", "20");
        student.put("grade", "A");
        
        // Получение значений
        System.out.println(student.get("name"));  // Alice
        
        // Проверка наличия ключа
        if (student.containsKey("age")) {
            System.out.println("Возраст есть");
        }
        
        // Удаление
        student.remove("grade");
        
        // Перебор всех элементов
        for (String key : student.keySet()) {
            System.out.println(key + ": " + student.get(key));
        }
    }
}
